R/pkgs/report_generation/R/ReportBuildUtils.R

```diff
@@ -1,1112 +1,1679 @@
+                                        est = conv_round(pIncidICU_final),
+                                        lvl = paste0("peak inc ICU", pdeath)) %>%
+                                 select(lvl, est, ci, pdeath) %>% arrange(pdeath)) %>%
+                     bind_rows(xx %>%
+                                 mutate(ci = make_CI(nCurrICU_lo, nCurrICU_hi),
+                                        est = conv_round(nCurrICU_final),
+                                        lvl = paste0("peak ICU cap", pdeath)) %>%
+                                 select(lvl, est, ci, pdeath) %>% arrange(pdeath)) %>%
+                     bind_rows(xx %>%
+                                 mutate(ci = make_CI(nIncidDeath_lo, nIncidDeath_hi),
+                                        est = conv_round(nIncidDeath_final),
+                                        lvl = paste0("total inc death", pdeath)) %>%
+                                 select(lvl, est, ci, pdeath) %>% arrange(pdeath))
+  )
+}
+
+
+tlabels <- c(" ", "IFR")
+nlabels <- c("name", "pdeath", "est", "ci")
+
+for(i in 1:length(table_dates)){
+  tlabels <- c(tlabels,
+               paste0(print_pretty_date_short(table_dates[i]), "\nmean    "),
+               "\n    95% PI")
+  if(i>1){nlabels <- c(nlabels, paste0("est", i-1), paste0("ci", i-1))}
+}
+names(tlabels) <- nlabels
+
+flextable::flextable(tmp[,nlabels]) %>%
+  flextable::set_header_labels(values=tlabels) %>%
+  flextable::valign(valign="bottom") %>%
+  flextable::colformat_num(digits=0) %>%
+  flextable::autofit()
+
+}
+
+
+
+##'
+##' Make statewide table of infections, hosp, ICU, deaths, vents for given scenario
+##'
+##' @param current_scenario text string of scenario label for which to build table
+##' @param hosp_state_totals totals for hospitalization related data for state for all pdeath
+##' @param table_dates formatted table_dates object
+##' @param pdeath_labels pdeath formatted labels
+##' @param pdeath_filecode pdeath file column codes
+##'
+##' @return state scenario table
+##'
+##' @export
+##'
+make_scn_state_table_withVent <- function(current_scenario,
+                                 hosp_state_totals,
+                                 table_dates,
+                                 pdeath_labels,
+                                 pdeath_filecode){
+  
+  ci_lo = 0.025
+  ci_hi = 0.975
+  
+  if (length(pdeath_filecode)==1) {
+    stop("Currently does not support single values of pdeath")
+  }
+  
+  tmp <- data.frame(name=c("INFECTIONS",
+                           "HOSPITALIZATIONS\n  total", "", "",
+                           "  daily peak admissions", "", "",
+                           "  daily peak capacity", "", "",
+                           "ICU \n  total", "", "",
+                           "  daily peak admissions", "", "",
+                           "  daily peak capacity", "", "",
+                           "Ventilations \n total", "", "",
```

Comment:
minor but should be fixed to be VENTILATIONS

Reply:
fixed in dataseed_merge